index.html

```diff
@@ -141,7 +144,7 @@ <h3>
           practice would generally not be recommended, otherwise.
         </p>
         <pre class="HTML example" title="Redundant role on list">
-          &lt;!-- Avoid doing this! -->
+          &lt;!-- Generally avoid doing this! -->
```

Comment:
Why "generally"?

Reply:
the reason is indicated in the paragraph prior to this.